### PR TITLE
Update `Divider`'s `direction` values from `horizontal`/`vertical` to `block`/`inline`

### PR DESCRIPTION
**Pull request by billfienberg**
### Background

(Provide a link to any relevant issues AND provide a TLDR of the issue in this pull request)

### Solution

(Describe your solution, why this approach was chosen, and what the alternatives/impacts may be)

### 🎩

- ...

### Checklist

- [ ] I have :tophat:'d these changes
- [ ] I have updated relevant documentation

### CODE REVIEW

File: packages/ui-extensions/src/surfaces/admin/components/Divider/Divider.ts [hunk 1]
```diff
@@ -4,9 +4,9 @@ export interface DividerProps {
   /**
    * Specify the direction of the divider.
    *
-   * @defaultValue 'horizontal'
+   * @defaultValue 'block'
```
Comment (olavoasantos) [inline packages/ui-extensions/src/surfaces/admin/components/Divider/Divider.ts:7]: This should be `inline`

Comment (olavoasantos) [reply packages/ui-extensions/src/surfaces/admin/components/Divider/Divider.ts:7]: It's `inline`. If you check the implementation logic on the component, the default behaviour is `inline`.

Comment (billfienberg) [reply packages/ui-extensions/src/surfaces/admin/components/Divider/Divider.ts:7]:
```suggestion
   * @defaultValue 'inline'
```

Comment (billfienberg) [reply packages/ui-extensions/src/surfaces/admin/components/Divider/Divider.ts]: Fixed in e6b5b62622858f392e8fc6f53117cfc59f16ebff

**Pull request merged**